LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright (c) 2020-2022 Velexi Corporation
+   Copyright [yyyy] Velexi Corporation
```

Comment:
Should line 181 say to replace square brackets [] instead of curly brackets {}, and should this line say "Copyright [yyyy] [name of copyright owner]"?

Reply:
Thanks for catching these typos. I have made the fixes.